specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2023-02-01/bms.json

```diff
@@ -9666,7 +9666,7 @@
           }
         },
         "isSoftDeleteFeatureStateEditable": {
-          "description": "Is soft delete feature state editable",
+          "description": "This flag is no longer in use. Please use 'softDeleteFeatureState' to set the soft delete state for the vault",
```

Comment:
Is this a breaking chage for your customers ? You still need to honor this for backward compatibility right ?

Reply:
No @raosuhas, its not a breaking change. We are still keeping the flag for backward compatibility. We are merely changing the description.

Reply:
thanks for confirming